client/view/Docs/Docs.css

```diff
@@ -9,15 +9,16 @@
 
 .Docs section {
   padding: 16px;
+  margin: 0 -16px;
   background-color: var(--bg-highlited);
   border-radius: 8px;
 
   & + & {
     margin-top: 24px;
   }
 
-  @media (width <= 740px) {
-    margin: 0 -16px;
+  @media (width > 740px) {
+    margin: 0;
```

Comment:
Not sure about this change. We have more properties (2 instead of 1) and reset property which you need a context to understand.

Reply:
this one or all of them?)

Reply:
let's postpone this changes for the future)

maybe there will be another breakpoint or 2-3 components as `Interactive`

Reply:
Only this one. Other changes is really improving readability.

Reply:
> let's postpone this changes for the future)

nooo :) just revert this change and we will merge rest of them

Reply:
I can do only 2 commits: 
- Remove unused `flex-basis` and `max-height` from `Interactive`
- Put `@meida` in `:root`